src/extractor/myexperience/posting/documents.ts

```diff
@@ -0,0 +1,30 @@
+export type ApplicationDocument =
+  | "Cover letter"
+  | "Grade report"
+  | "Resume"
+  | "Student Information Summary (SIS)"
+  | "Transcript"
+  | "Work history"
+  | "Other"
+  | "UNKNOWN";
+
+export function parseApplicationDocument(
+  document: string
+): ApplicationDocument {
+  document = document
+    .toLowerCase()
+    .trim()
+    .normalize("NFD")
+    .replace(/[\u0300-\u036f]/g, "");
+  if (document.includes("cover") && document.includes("letter"))
+    return "Cover letter";
+  else if (document.includes("grade") && document.includes("report"))
+    return "Grade report";
+  else if (document.includes("work") && document.includes("history"))
+    return "Work history";
+  else if (document.includes("SIS")) return "Student Information Summary (SIS)";
+  else if (document.includes("Transcript")) return "Transcript";
```

Comment:
Comparisons will always fail since document is lower case (line 15)
```suggestion
  else if (document.includes("sis")) return "Student Information Summary (SIS)";
  else if (document.includes("transcript")) return "Transcript";
```